determined-ai/determined
# fix: show selections in ComparisonView on any page (ET-189)

## Ticket

ET-189


## Description

Even if selected experiments/runs are not loaded on the current page, the Compare panel should still fetch data for them.


## Test Plan

On Experiments List page (with "New Experiment List" feature flag on):
- Select an experiment
- Open the "Compare" view
- Change the page
- Should display "Metrics", "Hyperparameters", and "Details" tabs with appropriate content
  - Should NOT display "No items selected." message

On Runs List page (with "Flat Runs View" feature flag on):
- Select a run
- Open the "Compare" view
- Change the page
- Should display "Metrics", "Hyperparameters", and "Details" tabs with appropriate content
  - Should NOT display "No items selected." message

## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.




[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
